internal/pkg/cli/flag.go

```diff
@@ -165,6 +165,9 @@ Mutually exclusive with the --%s flag.`, inputFilePathFlag)
 Mutually exclusive with the --%s flag.`, inputFilePathFlag)
 	secretInputFilePathFlagDescription = fmt.Sprintf(`Optional. A YAML file in which the secret values are specified.
 Mutually exclusive with the -%s ,--%s and --%s flags.`, nameFlagShort, nameFlag, valuesFlag)
+
+	repoURLFlagDescription = fmt.Sprintf(`The repository URL to trigger your pipeline.
+Supported providers are: %s`, strings.Join(template.QuoteSliceFunc(manifest.PipelineProviders), ", "))
```

Comment:
nit:
```suggestion
Supported providers are: %s`, strings.Join(manifest.PipelineProviders, ", "))
```
I feel like it's okay to not quote the providers here. It would have been necessary if the usage is like:
```
Specify the provider name. The supported options are: "GitHub", "Bitbucket", and "CodeCommit"
```
where "GitHub", "Bitbucket", and "CodeCommit" are the exact strings that the user needs to enter.

Please let me know if you think otherwise ☺️!